README.md

```diff
@@ -458,6 +459,33 @@ Example hash of inserting headers into responses:
 }
 ```
 
+## response-messaging attributes
+
+* `node[ 'repose' ][ 'response-messaging' ][ 'status_codes' ] - Array of status codes to activate on and generate responses.
```

Comment:
You might want to clean up any `Space inside square brackets detected.` infractions that rubocop declares.  Even though this is a block in markdown, its consistent elsewhere.